src/services/ExecutionService.js

```diff
@@ -0,0 +1,12 @@
+import { Op } from "sequelize"
+import { ExecutionDAO } from "../models"
+
+export default class ExecutionService {
+
+  async createExecution(payload) {
+    payload.executions.forEach(function(itm){
+      itm.deviceToken = payload.deviceToken;
+     });
+      await ExecutionDAO.bulkCreate(payload.executions)
```

Comment:
Warnings do linter:
- usar uma _arrow function_, evitando `function`s não nomeadas;
- não mutar o argumento `itm`, recomendo substituir por um `map`.

```suggestion
    const executions = payload.executions.forEach((itm) => {
      return {
        ...itm,
        deviceToken: payload.deviceToken
      }
     });
      await ExecutionDAO.bulkCreate(executions)
```